### PR TITLE
Separate api and web docker images

### PR DESCRIPTION
**Pull request by PabloCastellano**
**IMPORTANT NOTICE**: docker image `blossomlabs/blobscan` is now deprecated in favor of [blossomlabs/blobscan-web](https://hub.docker.com/r/blossomlabs/blobscan-web/) and [blossomlabs/blobscan-api](https://hub.docker.com/r/blossomlabs/blobscan-api/)

Changes in environment variables:

* Removed: `BLOBSCAN_TAG`
* Added: `BLOBSCAN_WEB_TAG`, `BLOBSCAN_API_TAG`

This PR splits docker images into two: api and web. Until now we were using the same docker image for both apps. 

Additionally, it was not well optimized. Now the nextjs app is built using the standalone mode and the rest-api-server app is bundled using [ncc](https://github.com/vercel/ncc). Performance when running the new docker image is prominent and the images are much lighter.

Previous:

image

* [blossomlabs/blobscan](https://hub.docker.com/r/blossomlabs/blobscan/tags) == 1.1GB **(deprecated)**

After this PR:

* [blossomlabs/blobscan-web](https://hub.docker.com/r/blossomlabs/blobscan-web/tags) == 116MB **(new)**
* [blossomlabs/blobscan-api](https://hub.docker.com/r/blossomlabs/blobscan-api/tags) == 112.61MB **(new)**

Related documentation about docker optimization:

* https://nextjs.org/docs/pages/api-reference/next-config-js/output
* https://turbo.build/repo/docs/guides/tools/docker

## Still TODO

* [x] Make sure prisma command is installed in the image and is available for docker-entrypoint.sh
* [x] Swagger API UI is not working (js console says `ReferenceError: SwaggerUIBundle is not defined`)

### CODE REVIEW

File: packages/db/prisma/schema.prisma [hunk 1]
```diff
@@ -4,7 +4,7 @@
 generator client {
   provider        = "prisma-client-js"
   previewFeatures = ["tracing", "metrics"]
-  binaryTargets = ["native", "debian-openssl-3.0.x"]
+  binaryTargets   = ["native", "linux-arm64-openssl-3.0.x"]
```
Comment (PabloCastellano) [inline packages/db/prisma/schema.prisma:7]: please leave debian-openssl-3.0.x

Comment (PJColombo) [reply packages/db/prisma/schema.prisma:7]: Actually. It looks like the rest api build is not complaining anymore about these missing binaries. Maybe we can remove this line now as it adds an extra unused binary to the image 🤔 ?

**Comment by PJColombo** [comment]
Pushed some changes addressing the last TODOs items

**Review by PJColombo — APPROVED** [review]
LGTM!

Initially, I attempted to copy the Prisma dependencies, but I encountered issues due to missing Prisma engine files. I suppose I was copying them incorrectly.

It's great to know we can actually copy and use them this way.

**Pull request merged**